server/src/daemon/service.rs

```diff
@@ -286,7 +288,8 @@ impl Service {
     pub async fn new() -> Self {
         Self {
             collections: RwLock::new(Vec::new()),
-            keyring: Arc::new(Keyring::load_default().await.unwrap()),
+            keyring: Arc::new(Keyring::load(DEFAULT_KEYRING_PATH, todo!()).await.unwrap()),
```

Comment:
> Even load_default call is actually wrong. You need to call .load instead as you don't use the portal's key to encrypt the keyring.

Yeah, I agree that we should call `load` here. And `load` returns a new keyring if the one we request is not found: https://github.com/bilelmoussaoui/oo7/blob/5b73cfb7b90a93501b140f745586eb11b1c5332e/client/src/portal/mod.rs#L110

But, I'm really not sure how to create and pass the `Secret` argument to `load`. I think this is one of the places that we need the Prompt.

Reply:
Isn't the key something you get from tpm?